xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -287,6 +287,10 @@ bool CGUIWindowMusicNav::GetDirectory(const std::string &strDirectory, CFileItem
       OnRetrieveMusicInfo(items);
   }
 
+  //Navigating music files so default content to "songs" unless in sources folder.
+  //This content allows view type to include media info so that file tag data can be displayed
+  if (!URIUtils::IsSourcesPath(strDirectory))
```

Comment:
Proposed change should do the job. I've added two more places to ensure we're not using a wrong content type .. https://gist.github.com/mkortstiege/ea9840d76d6ae1998c13. On the long run we should try to clean this mess up ;)